docs/build-tools/maven.md

```diff
@@ -0,0 +1,61 @@
+---
+id: maven
+title: Maven
+---
+
+Maven is one of the most common build tools in the JVM ecosystem and it also allows for using scala through the [scala-maven-plugin](https://davidb.github.io/scala-maven-plugin/usage.html).
+
+## Automatic installation
+
+The first time you open Metals in a new workspace it prompts you to import the
+build. Select "Import build" to start automatic installation.
+
+This will create all the needed bloop files, however there will be a warning, since the semanticDB plugin was not added yet in the automatic import. This will be added later. Most features should work without it, however some require semanticDB files to be provided alongside compiled data. To do that we need a couple of steps that are explained in the manual instalation section.
```

Comment:
I recommend formatting markdown files with prettier even if we don't enforce this in CI. Prettier provides automatic line-wrapping which makes reviewing a bit easier.

Reply:
```suggestion
This will create all the needed Bloop files, however there will be a warning, since the SemanticDB plugin was not added yet in the automatic import. This will be added later. Most features should work without it, however some require SemanticDB files to be provided alongside compiled data. To do that we need a couple of steps that are explained in the manual installation section.
```